packages/freezed/lib/src/templates/properties.dart

```diff
@@ -9,6 +9,16 @@ import '../utils.dart';
 import 'concrete_template.dart';
 import 'prototypes.dart';
 
+const privConstUsedErrorString =
+    'It seems like you constructed your class using MyClass._()'
```

Comment:
Could we make `MyClass` the actual class name?

Reply:
Sure, just figured that it would be easier and more performant to have it like this but a method like this would also do.
```dart
String _errorMessage(String s) => "Text here $s text here";
```

Reply:
By looking at it it would over complicate a lot the code, having to pass the class name down the tree up to the `Getter ` class, plus I can't seem to be able to get a consistent name with it. We can just not mention the class if you want ?